src/main/java/seedu/address/model/person/Remark.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Person's remark in the address book.
+ * Guarantees: immutable; is always valid
+ */
+public class Remark {
+    public final String value;
+
+    /**
```

Comment:
Seems like there is no regex for this field, which makes sense since is just a remark of a person